R-packages/data-raw/geo/county/cb_2019_us_county_5m.cpg

```diff
@@ -0,0 +1 @@
+UTF-8
```

Comment:
What is this file for?

Reply:
These are the spatial polygon files provided by the US Census Bureau. I think they are in ArcGIS format (could be wrong about that though). I have no idea what each of the individual parts are for, but they are needed to read the data into R. I am putting them here in case at some point we need to re-generate the smaller geojson files that are in the actual package.

Reply:
Yeah, shapefiles are an archaic format that consists of a bunch of smaller files. Apparently this one is "An optional file that can be used to specify the codepage for identifying the characterset to be used."

https://desktop.arcgis.com/en/arcmap/10.3/manage-data/shapefiles/shapefile-file-extensions.htm